packages/accounts/src/msca/account/multisigAccount.ts

```diff
@@ -118,7 +121,14 @@ export async function createMultisigModularAccount({
 
   return {
     ...baseAccount,
+    getLocalThreshold: () => threshold,
     publicKey: await signer.getAddress(),
     getSigner: () => signer,
   };
 }
+
+export const isMultisigModularAccount = (
+  acct: SmartContractAccount
+): acct is MultisigModularAccount => {
+  return acct.source === "MultisigModularAccount";
+};
```

Comment:
somewhat... hahah the MultisigModularAccount methods set the source to this value. I think what I need to do is export it as a constant somewhere so that it's not copy pasted